tendrl/monitoring_integration/grafana/alert_dashboard.py

```diff
@@ -250,7 +250,7 @@ def check_duplicate(
         new_resource_flag = True
         integration_id = str(resource["integration_id"])
         for row in dashboard_json["dashboard"]["rows"]:
-            if "panels" in row:
+            if "panels" in row and new_resource_flag:
```

Comment:
What is use of this `new_resource_flag` as its always `true` ?

Reply:
When we find out dashboard panel for particular resource already present in dashboard json I am changing new_resource_flag = False, because we don't need to update existing dashboard json.  Once new_resource_flag is marked as False then we don't need to search further the same resource in dashboard json because it is already found. When it is false break the loop and continue with the next resource. 

This check will reduce number of loop execution.

Reply:
Ack